src/panel/direction/DirectionPanel.jsx

```diff
@@ -131,7 +131,8 @@ export default class DirectionPanel extends React.Component {
   }
 
   async setTextInput(which, poi) {
-    if (poi.type === 'latlon') {
+    if (!poi.address || Object.keys(poi.address).length === 0) {
```

Comment:
You can use the `isNullOrEmpty` util from `src/libs/object.js` to do that.

Reply:
great ! I was looking for an equivalent to `lodash.isEmpty()` :)